Makefile

```diff
@@ -29,62 +82,96 @@ run: generate fmt vet
 debug: generate fmt vet
 	dlv debug cmd/manager/main.go
 
+# Run go fmt against code
+fmt:
+	go fmt ./pkg/... ./cmd/...
+
+# Run go vet against code
+vet:
+	go vet ./pkg/... ./cmd/...
+
+## --------------------------------------
+## Deploying
+## --------------------------------------
+
 # Install CRDs into a cluster
-install: manifests
-	kubectl apply -f config/crds
+install: $(KUSTOMIZE)
+	kubectl apply -k config/crds
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
-deploy: manifests
-	kubectl apply -f config/crds
-	kustomize build config/default | kubectl apply -f -
+deploy: $(KUSTOMIZE)
+	$(KUSTOMIZE) build config/default | kubectl apply -f -
 
 
 # unDeploy controller in the configured Kubernetes cluster in ~/.kube/config
-undeploy: manifests
-	kustomize build config/default | kubectl delete -f -
-
-# Generate manifests e.g. CRD, RBAC etc.
-manifests: controller-gen
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:artifacts:config=config/crd/bases
+undeploy: $(KUSTOMIZE)
+	$(KUSTOMIZE) build config/default | kubectl delete -f -
 
+# Deploy wordpress
+deploy-wordpress: $(KUSTOMIZE)
```

Comment:
This is a wonderful thing. Thanks. Do you think we could add this to e2e test ?
If not now may be later.